Gems/Atom/Feature/Common/Code/Source/PostProcess/FilmGrain/FilmGrainSettings.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/RTTI/ReflectContext.h>
+
+#include <Atom/RHI.Reflect/ShaderResourceGroupLayoutDescriptor.h>
+
+#include <Atom/Feature/PostProcess/FilmGrain/FilmGrainSettingsInterface.h>
+#include <Atom/RPI.Public/Image/StreamingImage.h>
+#include <PostProcess/PostProcessBase.h>
+
+#include <Atom/RPI.Public/View.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        class PostProcessSettings;
+
+        class FilmGrainSettings final
```

Comment:
As part of having good documentation add a quick comment about the purpose of this class.